src/Map/tests/MapTest.php

```diff
@@ -24,7 +24,7 @@ class MapTest extends TestCase
     public function testCenterValidation(): void
     {
         self::expectException(InvalidArgumentException::class);
-        self::expectExceptionMessage('The center of the map must be set.');
+        self::expectExceptionMessage('The map "center" must be explicitly set when not enabling "fitBoundsToMarkers" feature.');
```

Comment:
I still think we should make it true per default 🙊

Reply:
😠

Reply:
More seriously, fit bounds to marker is not a feature from Google Maps or Leaflet, it's a feature from UX Map, and make it automatically enabled depending of certains conditions looks wrong to me, because it behave differently than GMaps/Leaflet in this situation.